src/utils/blockstamp.py

```diff
@@ -25,12 +25,15 @@ def build_blockstamp(
 def _build_blockstamp_data(
     slot_details: BlockDetailsResponse,
 ) -> dict:
-    execution_data = slot_details.message.body['execution_payload']
+    if "execution_payload" not in slot_details.message.body:
+        raise ValueError(f"ExecutionPayload not found for slot {slot_details.message.slot}")
+
+    execution_payload = slot_details.message.body["execution_payload"]
 
     return {
-        'slot_number': SlotNumber(int(slot_details.message.slot)),
-        'state_root': slot_details.message.state_root,
-        'block_number': BlockNumber(int(execution_data['block_number'])),
-        'block_hash': execution_data['block_hash'],
-        'block_timestamp': Timestamp(int(execution_data['timestamp']))
+        "slot_number": SlotNumber(int(slot_details.message.slot)),
```

Comment:
why `"`? looks like we use `'` everywhere

Reply:
953 `"` and 651 `'`, so no everywhere. And I've chosen the one used by black by default. AFAIR there was an intention to format the "src" someday? cc @F4ever you word, ser?

Reply:
I try to use everywhere `'`.
And yeah, there is a setting in black to format to `"`, but no setting to format everything to `'`